index.html

```diff
@@ -4683,82 +4683,134 @@ <h2>Internationalization Considerations</h2>
+      <h3>Language and Direction</h3>
 
       <p>
-When expressing human-readable text content in the data model, it is important
-for presentation, accessibility, and further processing to specify the language
-of the text, as follows:
+Data publishers are strongly encouraged to use the design pattern described
+in this section when expressing simple internationalized strings in
+JSON, JSON-LD, or similar data formats to ensure the broadest interoperability
+across syntaxes. The design pattern is a result of extensive collaboration
+between the W3C Internationalization Activity, the Linked Data Community, and
+developers that primarily utilize JSON and CBOR to publish information.
       </p>
 
-      <ul>
-        <li>
-[[!JSON-LD]] supports the expression of language information using the
-<code>@language</code> feature, as described in [[!JSON-LD]] 1.1,
-<a href="https://www.w3.org/TR/json-ld11/#string-internationalization">Section 4.2.3: String Internationalization</a>.
-        </li>
-        <li>
-[[!JSON-LD]] also supports the expression of language information using the
-<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]] 1.0,
+      <p>
+The general design pattern is to use the following markup template when
+expressing a text string that is tagged with a language and, optionally, a
+specific text direction:
+      </p>
+
+      <pre class="example nohighlight" title="Design pattern for internationalized text strings">
+"<a>property</a>": {
+  "value": "<span class="highlight">The string value</span>",
+  "language: "<code>LANGUAGE</code>-x-d-<code>DIRECTION</code>"
+}
+      </pre>
+
+      <p>
+An example utilizing the design pattern above is shown below to express the
+human-readable text "revoked" in the English language without specifying a
+text direction:
+      </p>
+
+      <pre class="example nohighlight" title="Internationalized English text">
+"statusReason": {
+  "value": "<span class="highlight">Revoked</span>",
+  "language: "<code>en</code>"
+}
+      </pre>
+
+      <p>
+The next example expresses the human-readable text "revoked" in the Arabic
+language using a right-to-left text layout:
+      </p>
+
+      <pre class="example nohighlight" title="Internationalized Arabic text with a right-to-left layout">
+"statusReason": {
+  "value": "<span class="highlight">إلغاء</span>",
+  "language: "<code>ar</code>-x-d-<code>rtl</code>"
+}
+      </pre>
+
+      <p class="note">
+Utilization of the design pattern above assumes that the
+The JSON-LD <a href="#extensibility">extension</a> associated with the
+<a>property</a> utilizes the <code>@scoped</code> feature of JSON-LD to
+alias the <code>@language</code> and <code>@value</code> keywords to
+<code>language</code> and <code>value</code>, respectively.
+      </p>
+
+    </section>
+
+    <section class="informative">
+      <h3>Multiple Languages</h3>
+
+      <p>
+When multiple languages are utilized in the same value, a more complex
```

Comment:
Still not keen on this subsection. Yes, it is sometimes necessary to provide substring language metadata. This is rarely something that can be done with normal string data, but if markup information exists, one benefit of it is allowing the HTML to be transmitted with the language (and direction) markup intact.

However, this text suggests that it `is required`, which is too strong in my opinion. I would downgrade this subsection to a note, citing the fact that if you use `rdf:HTML` you can transmit strings that have embedded language and direction markup for a richer text experience.

Note that many consumers of these strings just rip the HTML out so they can make the strings into plain text again (which is one argument for metadata).